lib.php

```diff
@@ -164,6 +164,38 @@ function theme_boost_union_get_extra_scss($theme) {
     // We have to accept this fact here and must not copy the code from theme_boost_get_extra_scss into this function.
     // Instead, we must only add additionally CSS code which is based on any Boost Union-only functionality.
 
+    // In contrast to Boost core, Boost Union should add the login page background to the body element as well.
+    // That's why we first have to revert the background which is set to #page on the login page by Boost Core already as soon as a
```

Comment:
Please write Core in Boost core in lower case consistently.

Reply:
Thanks, Luca, for this hint! This is changed now.